generated/nifake/nifake.proto

```diff
@@ -379,11 +379,12 @@ message GetAnIviDanceWithATwistByteArrayResponse {
 }
 
 message GetAnIviDanceWithATwistStringRequest {
```

Comment:
This extra session input parameter was added in the base hapigen metadata for ni-fake sometime after we started using it for grpc-device. So these updates are pulling in that update and updating the request fields appropriately.